app-tasks/rf/src/rf/uploads/geotiff/create_scenes.py

```diff
@@ -10,7 +10,7 @@
 logger = logging.getLogger(__name__)
 
 
-def create_geotiff_scene(tif_path, organizationId, datasource,
+def create_geotiff_scene(tif_path, organizationId, datasource, acquisitionDate=None, cloudCover=None,
```

Comment:
Yeah, end it with `Z` instead of `+00:00`. Sorry should have included the more finicky example.